opentelemetry-otlp/src/exporter/tonic/trace.rs

```diff
@@ -89,11 +89,12 @@ impl SpanExporter for TonicTracesClient {
         Ok(())
     }
 
-    fn shutdown(&mut self) -> OTelSdkResult {
-        match self.inner.take() {
-            Some(_) => Ok(()), // Successfully took `inner`, indicating a successful shutdown.
-            None => Err(OTelSdkError::AlreadyShutdown), // `inner` was already `None`, meaning it's already shut down.
-        }
+    fn shutdown(&self) -> OTelSdkResult {
+        // match self.inner.take() {
```

Comment:
need to make self mutable for this to work, shutdown for tonic needs to be implemented according to https://github.com/open-telemetry/opentelemetry-rust/issues/2777

Reply:
can you add a TODO here linking to the issue.